tutorials/understanding-group/index.md

```diff
@@ -0,0 +1,404 @@
+    "value": "IjEi",
+    "index": true
+  }
+]
+```
+
+Where `Z3JvdXBfaWQ=` is a [Base64 encoding](https://www.browserling.com/tools/base64-decode) of `group_id`, and `IjEi` is a Base64 encoding of `"1"`, including the `"`. Therefore your group ID is `1`. Or with a one-liner:
+
+```sh
+simd query tx 079D9B213DCDE99DB0E31A8AFE9B0FDC605C81C1880D08D99A493A7BC52FAC23 --output json | jq '.events' | jq -r '.[] | select(.type == "cosmos.group.v1.EventCreateGroup") | .attributes[0].value' | base64 --decode | jq -r '.'
+```
+
+Query and verify the group that you just created and its ID that you just extracted:
+
+```sh
+$ simd query group groups-by-admin $ALICE
+```
+
+This last command outputs `1` too. This shows you that the group and its `id` can be recalled. Use that `id` for querying the group members.
+
+```sh
+$ simd query group group-members $GROUP_ID
+```
+
+Nice! Your group has `best football association` as metadata (which you can recall with the `group-info` command), Alice as group admin, and Alice and Bob as group members.
+
+## Manage group members
+
+To update the group's members, you send a transaction using the `update-group-members` command and a JSON file modeled on the previous `members.json`. The file only needs to contain the changes to the membership. Unchanged members do not need to be included in the `members_updates.json`.
+
+To add a member to the group, you mention it in the JSON, and to remove a member from the group, you mention it and set this member's voting weight to `0`.
+
+Let's add Carol, Dave and Emma as group members and remove Bob. Create `members_update.json` with:
+
+```json
+{
+    "members": [
+        {
+            "address": "bobaddr", // $BOB
+            "weight": "0" // this deletes bob
+            // The metadata does not need to be mentioned
+        },
+        {
+            "address": "caroladdr",
+            "weight": "1",
+            "metadata": "treasurer"
+        },
+        {
+            "address": "daveaddr",
+            "weight": "1",
+            "metadata": "player"
+        },
+                {
+            "address": "emmaaddr",
+            "weight": "1",
+            "metadata": "player"
+        },
+    ]
+}
+```
+
+```sh
+$ simd tx group update-group-members $ALICE $GROUP_ID members_updates.json
+```
+
+You can verify that the group members are updated:
+
+```sh
+$ simd query group group-members $GROUP_ID
+```
+
+As an exercise, please add Bob back in the group and go to the next section.
+
+## Create a group policy
+
+Next you need to decide on a group policy. This defines how long a proposal can be voted on and how its outcome is calculated.
+Here you use the [`ThresholdDecisionPolicy`](https://github.com/cosmos/cosmos-sdk/blob/release/v0.46.x/proto/cosmos/group/v1/types.proto#L53-L62). It defines the threshold that the tally of weighted _yes_ votes must reach in order for a proposal to pass. Each member's vote is weighted by its weight as defined in the group.
```

Comment:
Is the proper word _group policy_ or _decision policy_ or _group decision policy_?

Reply:
Precised.